refinery/ui/source/js/dashboard/directives/data-set-preview.html

```diff
@@ -87,6 +124,7 @@
     <div
       class="singleStudy"
       ng-if="preview.numStudies === 1">
+      <div id="data-set-summary-summary">
```

Comment:
No. `data-set-summary` is the prefix for all IDs in this template to avoid conflicts and `-summary` identifies the summary section. People got confused by the term _data set preview_ that's why we phrased it _data set summary_ as it really doesn't preview the data but provides an overview or summary of the meta data. If you want we can prefix all IDs with _data-set-preview_